_press/18F-consulting.md

```diff
@@ -3,6 +3,6 @@ title: 18F Delivery and 18F Consulting
 permalink: /press/18F-consulting
 layout: bare
 ---
-[18F Delivery](/delivery) is the larger of our current service offerings. It is a collection of teams comprised of designers, developers, and product managers that work _for_ and _with_ other federal teams at agencies like the Federal Election Commission, the General Services Administration, and the Treasury ([among many others](https://18f.gsa.gov/dashboard/)) where they build tools to improve government services.
+[18F Delivery](/delivery) is the larger of our current service offerings. It's a collection of teams comprised of designers, developers, and product managers that work _for_ and _with_ other federal teams at agencies like the Federal Election Commission, the General Services Administration, and the Department of the Treasury ([among many others](https://18f.gsa.gov/dashboard/)) where they build tools to improve government services.
```

Comment:
@gboone @awfrancisco I'd change "where **they** build tools" to "where **we** build tools."